src/index.js

```diff
@@ -21,12 +21,12 @@ document.querySelector('.refresh').addEventListener('click', refresh);
 // ADD NEW TO-DO LIST
```

Comment:
- [ ] Please, consider removing comments from your code, to make it clear. Your code is understandable enough and does not need any comments - it is talking itself about what happening 👍

Reply:
Removed the same.